src/mame/machine/esqpanel.h

```diff
@@ -107,6 +112,24 @@ class esqpanel_device : public device_t, public device_serial_interface
 
 	esqpanel::external_panel_server *m_external_panel_server;
 
+	const std::string slashes_to_separators(const std::string s) const {
```

Comment:
I can't see why this is non-static.  Also, would utilities for conversion between native filesystem paths and canonical URL fragments be something you'd want in a common place, not in a driver?

Reply:
Made it static (and moved it into esqpanel.cpp, so that I can use the logmacro.h macros for logging).

Such utilities would be good in a common place, and are also part of C++17 in std::filesystem::path. For now, just having this here locally is probably sufficient for the immediate term, and is then also easy to refactor once this kind of functionality is used elsewhere.